src/include/wiredtiger.in

```diff
@@ -5549,7 +5557,7 @@ struct __wt_page_log_put_args {
        uint64_t backlink_checkpoint_id;
        uint64_t base_checkpoint_id;
 
-       char dek[32]; /* an encrypted version of the key used to encrypt data */
+       WT_PAGE_LOG_ENCRYPTION encryption;
```

Comment:
Unless the server is doing something very weird around `memcpy`ing entire `WT_PAGE_LOG_GET`/`PUT` structures, I assume this needs a server code change. That'll have to go in atomically with this, right?

Reply:
I put in the `char dek[32]` a couple weeks ago, but I don't think the server uses that field yet.  At least it's not in disagg-storage-services/mongod-v8.0

Reply:
Ah, I see. Thanks :-)